wvd-templates/Update existing WVD host pool/mainTemplate.json

```diff
@@ -268,7 +275,7 @@
         "vhds": "[concat('vhds','/', variables('rdshPrefix'))]",
         "subnet-id": "[resourceId(parameters('virtualNetworkResourceGroupName'),'Microsoft.Network/virtualNetworks/subnets',parameters('existingVnetName'), parameters('existingSubnetName'))]",
         "vmTemplateName": "[concat( if(variables('rdshManagedDisks'), 'managedDisks', 'unmanagedDisks'), '-', toLower(replace(parameters('rdshImageSource'),' ', '')), 'vm')]",
-        "vmTemplateUri": "[concat(parameters('_artifactsLocation'),'/nestedtemplates/',variables('vmTemplateName'),'.json')]",
+        "vmTemplateUri": "[concat(parameters('_artifactsLocation'), 'nestedtemplates/', variables('vmTemplateName'), '.json')]",
```

Comment:
Can you start vmTemplateUri with the backlash (in combination) with the '_artifactsLocation'?

Reply:
Ignore. Suggestions are already aligned with "Create".